### PR TITLE
Update release build with compliance tasks

### PR DESCRIPTION
**Pull request by daxian-dbw**
Update release build with compliance tasks

### CODE REVIEW

File: .vsts-ci/templates/compliance.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
+  displayName: 'Publish Security Analysis Logs to Build Artifacts'
+  continueOnError: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@1
+  displayName: 'TSA upload to Codebase: PSThreadJob_201912 Stamp: Azure'
+  inputs:
+    tsaStamp: $(TsaStamp)
+    codeBaseName: $(CodeBaseName)
```
Comment (adityapatwardhan) [inline .vsts-ci/templates/compliance.yml]: Also update the value of CodeBaseName variable

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Comment by daxian-dbw** [comment]
Validating the changes in a AzureDevOps build, will wait till it succeeds.

File: .vsts-ci/templates/compliance.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  displayName: 'Run CredScan'
+  inputs:
+    debugMode: false
+  continueOnError: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@3
+  displayName: 'Run BinSkim '
+  inputs:
+    InputType: Basic
+    AnalyzeTarget: '$(Build.SourcesDirectory)\PSThreadJob\bin\${{ parameters.configuration }}\${{ parameters.framework }}\*.dll'
```
Comment (lzybkr) [inline .vsts-ci/templates/compliance.yml]: PSThreadJob?

Comment (daxian-dbw) [reply .vsts-ci/templates/compliance.yml]: oops, copy-paste mistake ... will update soon.

**Comment by daxian-dbw** [comment]
@adityapatwardhan the Azure DevOps build failed because the defender scan took more than an hour and the build agent cancelled the task.

**Comment by daxian-dbw** [comment]
Increase the timeout to 200 minutes. Test build has been kicked off in Azure DevOps.

**Comment by daxian-dbw** [comment]
@adityapatwardhan The test build in Azure DevOps has passed. It took 1.5 hours, and the API scan itself took 1 hour 17 minutes :)

**Pull request merged**